server/audit_math/macro.py

```diff
@@ -304,7 +310,8 @@ def compute_risk(
         else:
             p *= ((1 - 1 / U) / (1 - taint)) ** times_sampled[batch]
 
-        if p <= alpha:
-            return float(p), True
+        # Make sure we're returning the lowest p-value in the sequence
+        if p <= low_p:
+            low_p = p
```

Comment:
Does this logic depend on the order of the batches in `sample_results`? It seems like the p-value gets multiplied each iteration of the loop, which means the order of the multiplication would determine what the lowest p-value is.

If so, I'm wondering what order we expect them to be in

Reply:
The order should be the order they were drawn.